db/migrate/20190621160800_rename_host_group_matchers.rb

```diff
@@ -1,20 +1,17 @@
 class RenameHostGroupMatchers < ActiveRecord::Migration[5.2]
   def up
-    host_group_matchers = Setting.find_by_name('host_group_matchers_inheritance')
-    matchers_inheritance = Setting.find_by_name('matchers_inheritance')
-    return unless host_group_matchers.present? && matchers_inheritance.present?
-    matchers_inheritance.update_attribute(
+    host_group_matchers= Setting.where(name: 'host_group_matchers_inheritance')
+    return unless host_group_matchers.exists?
+    Setting.where(name: 'matchers_inheritance').update_all(
       :value,
```

Comment:
`update_all` takes a hash, not an attribute name and value:
```suggestion
    Setting.where(name: 'matchers_inheritance').update_all(
      value:
```